velox/expression/ExprConstants.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace facebook::velox::expression {
```

Comment:
This works.

Have we decided how to split the code between velox::expression, velox::core and velox::exec? What would be the rules for deciding what goes where? 

It might be nicer to introduce a class, BuiltinNames or SpecialFormNames or similar. Or maybe, SpecialFormKind enum w/ VELOX_DEFINE_ENUM_NAME.